impls/ntt-data-on-premise.yaml

```diff
@@ -0,0 +1,165 @@
+            n_ports: 24
+            c_intensity: 457 # gCO2/kwh
+            embodied_c: 0.251 #g per unit
+            lifespan: 4 # years
+            pue: 1.5
+        observations: 
+          series:
+            timestamp: 2023-07-06T00:00
+            duration: 1
+            five-min-input-rate: 100
+            five-min-output-rate: 100
+          mapping:
+              units: hours
+
+  layer-2-switch:
+    model: default  
+    config:
+    children: 
+      switch-1:
+        common:
+            p_baseline: 22.8 # W Baseline power based on full port 100% traffic for Cisco Catalyst 9300
+            n_ports: 24
+            c_intensity: 457 # gCO2/kwh
+            embodied_c: 7.16 #g per unit
+            lifespan: 4 # years
+            pue: 1.5
+        observations: 
+          series:
+            timestamp: 2023-07-06T00:00
+            duration: 1
+            five-min-input-rate: 100
+            five-min-output-rate: 100
+          mapping:
+            units: hours
+      switch-2:
+        common:
+            p_baseline: 22.8 # W Baseline power based on full port 100% traffic for Cisco Catalyst 9300
+            n_ports: 24
+            c_intensity: 457 # gCO2/kwh
+            embodied_c: 7.16 #g per unit
+            lifespan: 4 # years
+            pue: 1.5
+        observations: 
+          series:
+            timestamp: 2023-07-06T00:00
+            duration: 1
+            five-min-input-rate: 100
+            five-min-output-rate: 100
+          mapping:
+              units: hours
+      switch-3:
+        common:
+            p_baseline: 22.8 # W Baseline power based on full port 100% traffic for Cisco Catalyst 9300
+            n_ports: 24
+            c_intensity: 457 # gCO2/kwh
+            embodied_c: 7.16 #g per unit
+            lifespan: 4 # years
+            pue: 1.5
+        observations: 
+          series:
+            timestamp: 2023-07-06T00:00
+            duration: 1
+            five-min-input-rate: 100
+            five-min-output-rate: 100
+          mapping:
+              units: hours
+      switch-4:
+        common:
+            p_baseline: 22.8 # W Baseline power based on full port 100% traffic for Cisco Catalyst 9300
+            n_ports: 24
+            c_intensity: 457 # gCO2/kwh
+            embodied_c: 7.16 #g per unit
+            lifespan: 4 # years
+            pue: 1.5
+        observations: 
+          series:
+            timestamp: 2023-07-06T00:00
+            duration: 1
+            five-min-input-rate: 100
+            five-min-output-rate: 100
+          mapping:
+            units: seconds
+  db-servers:
+    model: default  
+    config:
+    children: 
+        common:
+          c_intensity: 457 # gCO2/kwh
+          embodied_c: 6270 #g per unit
+          lifespan: 4 # years
+          pue: 1.5
+        observations: 
+        series:
+          timestamp: 2023-07-06T00:00
+          duration: 1
+          instantaneous-power: 176
```

Comment:
- There is an embodied model here same as the other examples.
- There is a different energy model used here though, this takes as input observations of average power during the observation interval and outputs a kWh (energy value). So effectively a simple power (kW) -> energy (kWh) model..